_posts/2019-07-10-cockpit-198.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Cockpit 198
+author: pitti
+date: 2019-07-10
+tags: cockpit linux
+slug: cockpit-198
+category: release
+summary: Cockpit with PatternFly 4 user interface design update
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 198.
+
+### PatternFly4 user interface design
+
+Cockpit got restyled to match the
+[PatternFly 4](https://pf4.patternfly.org/components/Button/examples/)
+User Interface design, including the
+[Red Hat Text and Display fonts](https://github.com/RedHatOfficial/RedHatFont).
+
+This aligns Cockpit with other web user interfaces that use
+[PatternFly](https://www.patternfly.org/), such as
+[OpenShift 4](https://www.redhat.com/en/openshift-4).
+
+Over time, Cockpit will be ported to actually use PatternFly 4 widgets, but
+this restyle allows us to do this gradually.
+
+![login page](/images/login-pf4.png)
+
+![system page](/images/system-pf4.png)
+
+### SELinux: Show changes
+
+The SELinux page now has a new section "System Modifications" which shows all
+policy settings that were made to the system (with Cockpit or otherwise):
+
+![SELinux modifications](/images/selinux-modifications.png)
+
+The "View Automation Script" link will show a dialog with a shell script that
+can be used to apply the same changes to other machines:
+
+![SELinux automation script](/images/selinux-autoscript.png)
+
+### Machines: Deletion of Virtual Networks
+
+The Virtual Networks section of the Machines page now supports deleting networks.
+
+![delete virtual network](/images/machines-delete-network.png)
+
+### Machines: Support more disk types
+
+When creating a new VM, the disk can now be on a storage pool type other than a
+plain file. The newly supported types are iSCSI, LVM, and physical volumes.
+
+![VM creation with iSCSI pool](/images/machines-create-iscsi.png)
+
+### Docker: Change menu label
+
+The menu label changed from "Containers" to "Docker Containers". This avoids
+confusion with the "Podman Containers" page from
+[cockpit-podman](https://github.com/cockpit-project/cockpit-podman), and points
+out that this page really is about Docker only, not any other container
+technology.
+
+### Web server: More flexible https redirection for proxies
+
+[cockpit-ws](https://cockpit-project.org/guide/latest/cockpit-ws.8.html) now
+supports redirecting unencrypted http to https (TLS) even when running in
+`--no-tls` mode. Use this when running cockpit-ws behind a reverse http proxy
+that also supports https, but does no redirection from http to https for itself.
```

Comment:
You marked this resolved, but didn't change anything as far as I can tell?

Reply:
I replaced "by itself" with "for itself", that's how I understood your remark?